hospitalco/market/serializers.py

```diff
@@ -1,15 +1,25 @@
 from rest_framework import serializers
 
-from .models import Item, Requirement
+from .models import Category, Item, Requirement
 
 
 class ItemSerializer(serializers.ModelSerializer):
+    category = serializers.PrimaryKeyRelatedField(read_only=True)
+
     class Meta:
         model = Item
-        fields = ["id", "name", "details"]
+        fields = ["id", "name", "details", "category"]
```

Comment:
Me and @akshaytgithub  were of the opinion that `cateegory` field isn't probably required now

Reply:
It's not serialized in a nested manner, just the category ID turns up:
```
{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "id": 3,
            "name": "n95",
            "details": "life saver!",
            "category": 1
        }
    ]
}
```
Is that too much?